<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAndReplaceFormClass</class>
 <widget class="QWidget" name="FindAndReplaceFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FindAndReplaceForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    color: rgb(250, 250, 250);
    background-color: rgb(31, 31, 31);
}

QToolBar{
    color: rgb(250, 250, 250);
    background-color: rgb(31, 31, 31);
    border:none;
    min-height:18px;
	spacing: 3px;
}

QToolBar::handle {
    background-color: rgb(61, 61, 61);
}

QToolButton{
    color: rgb(250, 250, 250);
    background-color: rgb(31, 31, 31);
    border:none;
    min-height:20px;
}

QToolButton::hover{
    color: rgb(250, 250, 250);
    background-color: rgb(61,61,61);
    border: 1px solid rgb(112,112,112);
    min-height:20px;
}

QToolButton:pressed,QToolButton:checked {
    color: rgb(250, 250, 250);
    background-color: rgb(61,61,61);
    border: 1px solid rgb(46,46,46);
    min-height:20px;
}

QStatusBar{
    color: rgb(250, 250, 250);
    background-color: rgb(31, 31, 31);
    border:none;
}

QStatusBar::item {
     border: 0px solid red;
}

QMenuBar{
    color: rgb(214,214,214); 
    background-color: rgb(31,31,31)
}

QMenuBar::item {
    spacing: 0px;
    margin-top: 1px;
    margin-left: 1px;
    padding: 3px 8px;
    color: rgb(214,214,214); 
    background: transparent;
    border-radius: 0px;
    width: 65px;
    height: 26px;
}

QMenuBar::item:selected{
    border:1px solid rgb(112,112,112);background-color:rgb(61,61,61);color:rgb(250,250,250);
}

QMenu{background-color:rgb(46, 46, 46); padding:5px; border: 1px solid rgb(66,66,66);}
QMenu::item{color:rgb(214,214,214);padding-left:8px;padding-top:7px;padding-bottom:8px;padding-right:10px;background-color:rgb(46, 46, 46);}
QMenu::item:selected{border:1px solid rgb(112,112,112);color:rgb(250,250,250);background-color:rgb(61,61,61);}
QMenu::item:hover{border:1px solid rgb(112,112,112);background-color:rgb(250,250,250);}
QMenu::item:disabled{background:rgb(46, 46, 46); color:rgb(92,92,92);}
QMenu::QAbstractItemView{color:rgb(214,214,214);padding-left:8px;padding-top:7px;padding-bottom:8px;padding-right:10px;background-color:rgb(46, 46, 46);}

QToolTip{border-style:none; background-color: rgb(57, 58, 60);color: rgb(208, 208, 208);}

QWidget{
    color: rgb(214,214,214);
    background: rgb(31,31,31);
}

QDockWidget {
	color: rgb(214,214,214);
	background: rgb(31,31,31);
    /*titlebar-close-icon: url(:/image/icon_close.svg);
    titlebar-normal-icon: url(:/image/icon_pin.svg);*/
	border-bottom: 1px solid rgb(61,61,61);
	border-left: 1px solid rgb(61,61,61);
	border-right: 1px solid rgb(61,61,61);
}

QDockWidget::title {
	text-align: left;
    background: rgb(31,31,31);
    border-top: 1px solid rgb(61,61,61);
	border-left: 1px solid rgb(61,61,61);
	border-right: 1px solid rgb(61,61,61);
	font-size: 18pt;
}

QDockWidget::close-button:hover, QDockWidget::float-button:hover {
    background: rgb(214,214,214);
}

QDockWidget::close-button:pressed, QDockWidget::float-button:pressed {
	padding: 1px -1px -1px 1px;
    background: rgb(214,214,214);
}

QDockWidget::close-button {
	border-top: 1px solid rgb(61,61,61);
    background: rgb(31,31,31);
    icon-size:18px; /* maximum icon size */
    width: 20px;
	height:20px;
}

QDockWidget::float-button {
	border-top: 1px solid rgb(61,61,61);
    background: rgb(31,31,31);
    icon-size: 18px; /* maximum icon size */
    width: 20px;
	height:20px;
}

QTabWidget::pane{
    top:0px;
    border-top:2px solid #7160E8;
    border-bottom:1px solid rgb(61,61,61);
    border-right:1px solid rgb(61,61,61);
    border-left:1px solid rgb(61,61,61);
}

QTabWidget::tab-bar
{
    alignment: left;
    top:0px;
    left:5px;
    border:none;
}

QTabBar::tab:first{
    color:rgb(178,178,178);
    background:rgb(46,46,46);

    padding-left:0px;
    padding-right:0px;
    width:100px;
    height:24px;
    margin-left:0px;
    margin-right:0px;

    border-top:2px solid rgb(31, 31, 31);
}

QTabBar::tab:middle{
    color:rgb(178,178,178);
    background:rgb(46,46,46);
    padding-left:0px;
    padding-right:0px;
    width:100px;
    height:24px;
    margin-left:0px;
    margin-right:0px;

    border-top:2px solid rgb(31, 31, 31);

}

QTabBar::tab:last{
    color:rgb(178,178,178);
    background:rgb(46,46,46);

    padding-left:0px;
    padding-right:0px;
    width:100px;
    height:24px;
    margin-left:0px;
    margin-right:0px;

    border-top:2px solid rgb(31, 31, 31);
}

QTabBar::tab:selected{
    color:rgb(250,250,250);
    background:rgb(61,61,61);
    border-top:2px solid #7160E8;
}

QTabBar::tab:hover{
    color:rgb(250,250,250);
    background:rgb(61,61,61);
    padding-left:0px;
    padding-right:0px;
    width:100px;
    height:24px;
    margin-left:0px;
    margin-right:0px;
}

QTableWidget::item {
    color: rgb(214,214,214);
    background: rgb(31,31,31);
	min-height: 20px;
}

QTableWidget::item {
    color: rgb(214,214,214);
    background: rgb(31,31,31);
	min-height: 20px;
}

QTableWidget::item:selected {
    color: rgb(214,214,214);
    background:rgb(113,96,232);
}

QTableWidget::item:hover {
    background:rgb(61,61,61);
}

QTableWidget {
    color: rgb(214,214,214);
    background: rgb(31,31,31);
    border-top: 1px solid rgb(61,61,61);
	border-left: 1px solid rgb(61,61,61);
	border-right: 1px solid rgb(61,61,61);
	border-bottom: 1px solid rgb(61,61,61);
    alternate-background-color: #525252;
    gridline-color: transparent;
	font-size:12px;
	outline:none;
}

QTreeWidget {
    color: rgb(214,214,214);
    background: rgb(31,31,31);
    border-top: 1px solid rgb(61,61,61);
	border-left: 1px solid rgb(61,61,61);
	border-right: 1px solid rgb(61,61,61);
	border-bottom: 1px solid rgb(61,61,61);
    alternate-background-color: #525252;
    gridline-color: transparent;
	font-size:12px;
	outline:none;
}

QTreeWidget::branch{
    background: rgb(31,31,31);
}

QTreeWidget::branch:closed:has-children:!has-siblings,
QTreeWidget::branch:closed:has-children:has-siblings {
    border-image: none;
    image: url(:/image/icon_scrollbararrowcollapsed.svg);
}

QTreeWidget::branch:open:has-children:!has-siblings,
QTreeWidget::branch:open:has-children:has-siblings  {
    border-image: none;
    image: url(:/image/icon_scrollbararrowdownright.svg);
}

QTreeWidget::item {
    color: rgb(214,214,214);
    background: rgb(31,31,31);
	min-height: 20px;
}

QTreeWidget::item:selected {
    color: rgb(214,214,214);
    background:rgb(113,96,232);
}

QTreeWidget::item:hover {
    background:rgb(61,61,61);
}

QHeaderView {
    text-align: center;
    background: rgb(31,31,31);
    padding: 0px;
    margin: 0px;
    color: rgb(250,250,250);
    border-top: 0px solid rgb(61,61,61);
	border-left: 1px solid rgb(61,61,61);
	border-right: 1px solid rgb(61,61,61);
}

QHeaderView::section::vertical {
    text-align: center;
    background: rgb(31,31,31);
    padding: 3px;
    margin: 0px;
    color: rgb(250,250,250);
    border: 0px solid rgb(61,61,61);
    border-left-width: 0px;
	border-bottom-width: 1px;
	border-right-width: 1px;
}

QHeaderView::section::horizontal{
    text-align: center;
    background: rgb(31,31,31);
    padding: 3px;
    margin: 0px;
    color: rgb(250,250,250);
    border: 0px solid rgb(61,61,61);
    border-left-width: 0px;
	border-bottom-width: 1px;
	border-right-width: 1px;
}

QHeaderView::down-arrow {
    image: url(:/image/icon_sortascending.svg);
    width: 18px;
    height: 18px;
}
QHeaderView::up-arrow {
    image: url(:/image/icon_sortdescending.svg);
    width: 18px;
    height: 18px;
}

QScrollBar:vertical {
	min-height: 30px;  
    background: rgb(46,46,46);
}

QScrollBar::handle:vertical {
    background: rgb(71,71,71);
  	border: 4px solid  rgb(46,46,46);
	min-height:50px;
}

QScrollBar::handle:hover:vertical,
QScrollBar::handle:pressed:vertical {
    background: rgb(153,153,153);
}

QScrollBar::sub-page:vertical {
    background:transparent
}

QScrollBar::add-page:vertical {
    background:transparent
}

QScrollBar::add-line:vertical {
    background:transparent
}

QScrollBar::sub-line:vertical {
    background:transparent
}

QScrollBar:horizontal {
    background: rgb(46,46,46); 
}

QScrollBar::handle:horizontal {
    background: rgb(71,71,71);
  	border: 3px solid  rgb(46,46,46);
	min-width:50px;
}

QScrollBar::handle:hover:horizontal,
QScrollBar::handle:pressed:horizontal {
    background: rgb(153,153,153);
}

QScrollBar::sub-page:horizontal {
    background: 444444;
}

QScrollBar::add-page:horizontal {
    background: 5B5B5B;
}

QScrollBar::add-line:horizontal {
    background: none;
}

QScrollBar::sub-line:horizontal {
    background: none;
}

QToolTip{
    border-style:none; 
    background-color: rgb(57, 58, 60);
    color: rgb(208, 208, 208);
}

QListView {
    /*color: rgb(214,214,214);*/
    background: rgb(31,31,31);
    border: 1px solid rgb(61,61,61);
    alternate-background-color: #525252;
    gridline-color: transparent;
	outline:none;
}

QListView QTableCornerButton::section {
    /*color: rgb(214,214,214);*/
    background: rgb(31,31,31);
    border: 0px solid #418bc9;/*边框*/
    border-radius:0px;/*边框圆角*/
    border-color: rgb(41, 139, 201);/*边框颜色*/
    font: bold 11pt;/*字体大小*/
    padding:12px 0 0 10px;/*内边距*/
 }

QListView::item {
    /*color: rgb(214,214,214);*/
    background: rgb(31,31,31);
}

QListView::item:selected {
    /*color: rgb(214,214,214);*/
    background:rgb(61,61,61); /* qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #484848, stop:1 #383838);*/
}

QListView::item:hover {
    background:rgb(61,61,61);
}

/* 未下拉时，QComboBox的样式 */
QComboBox {
    border: 1px solid #424242;   /* 边框 */
    border-radius: 0px;   /* 圆角 */
    /*padding: 1px 13px 1px 3px;    字体填衬 */
    color: rgb(250,250,250);
    /*font: normal normal 15px &quot;Microsoft YaHei&quot;*/;
    background:#383838;
}

QComboBox:hover {
    border: 1px solid #424242;   /* 边框 */
    border-radius: 0px;   /* 圆角 */
    /*padding: 1px 18px 1px 3px;    字体填衬 */
    color: rgb(250,250,250);
    /*font: normal normal 15px &quot;Microsoft YaHei&quot;*/;
    background:#3D3D3D;
}

QComboBox:selected {
    border: 1px solid #999999;   /* 边框 */
    border-radius: 0px;   /* 圆角 */
    /*padding: 1px 18px 1px 3px;    字体填衬 */
    color: rgb(250,250,250);
    /*font: normal normal 15px &quot;Microsoft YaHei&quot;*/;
    background:#3D3D3D;
}
 
/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 0px solid #1F1F1F;   /* 选定项的虚框 */
    border: 1px solid #424242;    /* 整个下拉窗体的边框 */
    color: #FAFAFA;
    background-color: #2E2E2E;   /* 整个下拉窗体的背景色 */
    selection-background-color: #3D3D3D;   /* 整个下拉窗体被选中项的背景色 */
}
 
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
    height: 15px;  /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	border: 0px solid #707070;
	background-color: #2E2E2E;
}
 
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    color: #FAFAFA;
	border: 1px solid #707070;
    background-color:  #3D3D3D;   /* 整个下拉窗体越过每项的背景色 */
}
 
/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
    color: #FAFAFA;
	/*border: 1px solid #707070;*/
    background-color:  #3D3D3D; 
}


QComboBox QAbstractScrollArea QScrollBar:vertical {
    background: #2E2E2E;
	width:  15px;
	min-height: 60px;  
}

QComboBox QAbstractScrollArea QScrollBar::handle:vertical {
	background: #4D4D4D;
  	border: 3px solid  #2E2E2E;
	width:  15px;
	min-height: 60px;  
}

QComboBox QAbstractScrollArea QScrollBar::handle:hover:vertical,
QComboBox QAbstractScrollArea QScrollBar::handle:pressed:vertical {
    background: #999999; 
}

QComboBox QAbstractScrollArea QScrollBar::sub-page:vertical {
    background: 444444;
}

QComboBox QAbstractScrollArea QScrollBar::add-page:vertical {
    background: 5B5B5B;
}

QComboBox QAbstractScrollArea QScrollBar::add-line:vertical {
    background: none;
}

QComboBox QAbstractScrollArea QScrollBar::sub-line:vertical {
    background: none;
}
 

 
/* 设置为可编辑（setEditable(true)）editable时，编辑区域的样式 */
QComboBox:editable {
    background: #383838;
}
 
/* 设置为非编辑（setEditable(false)）!editable时，整个QComboBox的样式 */
QComboBox:!editable {
    background:#383838;
}
 
/* 设置为可编辑editable时，点击整个QComboBox的样式 */
QComboBox:editable:on {
    background:#383838;
    border: 1px solid #999999;   /* 边框 */
}
 
/* 设置为非编辑!editable时，点击整个QComboBox的样式 */
QComboBox:!editable:on {
    background:#383838;
}
 
/* 设置为可编辑editable时，下拉框的样式 */
QComboBox::drop-down:editable {
    background:#383838;
}
 
/* 设置为可编辑editable时，点击下拉框的样式 */
QComboBox::drop-down:editable:on {
    background:#383838;
}
 
/* 设置为非编辑!editable时，下拉框的样式 */
QComboBox::drop-down:!editable {
    background: #383838;
}
 
/* 设置为非编辑!editable时，点击下拉框的样式 */
QComboBox::drop-down:!editable:on {
    background:#383838;
}
 
/* 点击QComboBox */
QComboBox:on {
}
 
/* 下拉框样式 */
QComboBox::drop-down {
    subcontrol-origin: padding;   /* 子控件在父元素中的原点矩形。如果未指定此属性，则默认为padding。 */
    subcontrol-position: top right;   /* 下拉框的位置（右上） */
    width: 15px;   /* 下拉框的宽度 */
 
    border-left-width: 0px;   /* 下拉框的左边界线宽度 */
    border-left-color: #383838;   /* 下拉框的左边界线颜色 */
    border-left-style: solid;   /* 下拉框的左边界线为实线 */
    border-top-right-radius: 0px;   /* 下拉框的右上边界线的圆角半径（应和整个QComboBox右上边界线的圆角半径一致） */
    border-bottom-right-radius: 0px;   /* 同上 */
}

/* 越过下拉框样式 */
QComboBox::drop-down:hover {
    background:#3D3D3D;
}

/* 下拉箭头样式 */ 
QComboBox::down-arrow {
    width: 15px; /* 下拉箭头的宽度（建议与下拉框drop-down的宽度一致） */ 
    background: transparent; /* 下拉箭头的的背景色 */ 
    padding: 0px 0px 0px 0px; /* 上内边距、右内边距、下内边距、左内边距 */ 
    image: url(:/image/icon_down_arrow.png);
} 

/* 点击下拉箭头 */
QComboBox::down-arrow:on {
    image: url(:/image/icon_up_arrow.png);
}

QLineEdit{
    color:rgb(214,214,214);
    font-size:12px;
    border:1px solid rgb(66,66,66);
    background: rgb(56,56,56);
    background-image: url(:image/icon_search.svg);
    background-repeat: no-repeat;
    background-position: left;
    padding: 2 2 2 20;
}

QLineEdit::hover{
    color:rgb(250,250,250);
    font-size:12px;
    border:1px solid rgb(153,153,153);
    background: rgb(31,31,31);
}

QTextBrowser{
    color: rgb(214,214,214);
    background: rgb(37,37,38);
    border: 1px solid rgb(61,61,61);
}

QPushButton {
    background-color: #3D3D3D;
    border: 0px solid #dcdfe6;
    padding: 3px;
    color:rgb(214, 214, 214);
}

QPushButton:hover {
    background-color: #4D4D4D;
    color:rgb(250, 250, 250);
}

QPushButton:pressed, QPushButton:checked {
    background-color: #333333;
    color:rgb(250, 250, 250);
}

QSpinBox {
    border:1px solid #3D3D3D;
    color: #FAFAFA;
    background: #333333;
    selection-color: #FFFFFF;
    selection-background-color: #0078D7;
    font-size: 12pt;
}

QSpinBox:hover {
    color: #FAFAFA;
    background: rgb(61,61,61);
    border:1px solid #424242;
}

QSpinBox::up-arrow {
    image: url(:/image/icon_glyphup.svg);
    width: 14px;
    height: 14px;
}

QSpinBox::down-arrow {
    image: url(:/image/icon_glyphdown.svg);
    width: 14px;
    height: 14px;
}

QSpinBox::down-button, QSpinBox::up-button {
    background: #333333;
}


QSpinBox::up-button:hover, QSpinBox::down-button:hover{
    background: rgb(61,61,61);
}

QSpinBox::up-button:pressed, QSpinBox::down-button:pressed {
    background: rgb(91,91,91);
}

QCheckBox {
	color:white;
    spacing: 1px;
}

QCheckBox:disabled {
	color:gray;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
}

QCheckBox::indicator:unchecked {
	image: url(:/image/icon_unchecked_dark.svg);
}

QCheckBox::indicator:unchecked:hover {
	image: url(:/image/icon_unchecked_dark_hover.svg);
}

QCheckBox::indicator:unchecked:pressed {
	image: url(:/image/icon_unchecked_dark.svg);
}

QCheckBox::indicator:checked {
	image: url(:/image/icon_checked_dark.svg);
}

QCheckBox::indicator:checked:hover {
	image: url(:/image/icon_checked_dark_hover.svg);
}

QCheckBox::indicator:checked:pressed {
	image: url(:/image/icon_checked_dark.svg);
}

QCheckBox::indicator:indeterminate:pressed {
	image: url(:/image/icon_unchecked_dark.svg);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_keyword">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_replacekeyword">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_casesensitive">
       <property name="text">
        <string>Case-sensitive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_wholewordmatch">
       <property name="text">
        <string>Whole word match</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_regularexpression">
       <property name="text">
        <string>Use regular expression</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_findprevious">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Find Previous</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_findnext">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>FindNext</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_findall">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: #403582;
border: 0px solid #dcdfe6;
padding: 3px;
color:white;
}

QPushButton:hover {
background-color: #4D4D4D;
color:rgb(250, 250, 250);
}

QPushButton:pressed, QPushButton:checked {
background-color: #333333;
color:rgb(250, 250, 250);
}</string>
         </property>
         <property name="text">
          <string>Find All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_replaceall">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Replace All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
